truss/templates/training/job.py

```diff
@@ -28,13 +28,20 @@ def _signature_accepts_kwargs(signature: inspect.Signature) -> bool:
     return False
 
 
-def _add_bundled_packages_to_path(config):
+def _add_bundled_packages_to_path(config: dict):
     if "bundled_packages_dir" in config:
         bundled_packages_path = Path("/packages")
```

Comment:
@pankajroark why is this hardwired? shouldn't it pull the dir from the config?

Reply:
Seems like a mistake, good catch, could you please fix. Thanks